content/dev/dev-guide.md

```diff
@@ -102,79 +56,74 @@ A desk is a series of numbered commits, the most recent of which represents the
 For building applications to distribute on Urbit, we'll want to work on a new desk.
 
 #### Desk basics
+##### Setting up a new desk
+###### With Citadel
+The Citadel app allows you to create and manipulate your desks purely from the dojo. It provides a repl and a split screen front end that works with the docs app.
+ >  (You might be reading this on urbit already - skip the following if so)
+- Install on the livenet from `~mipber/citadel`, either via the UI or by typing `|install ~mipber %citadel` into the dojo. Once it finishes, type `|link %citadel-cli` to switch to a repl with custom commands reached through the `;` prefix. These allow you to create desks from examples or simply use another desk as a starting point.
```

Comment:
When I try to do this locally I can't install either of these. In the UI it can find the star name as a dev source but no packages on either of them.